README.md

```diff
@@ -0,0 +1,41 @@
+[![Build Status][ci-img]][ci] [![Released Version][maven-img]][maven]
+
+# OpenTracing Spring RabbitMQ
+
+This repository provides OpenTracing instrumentation for RabbitMQ. It can be used with any OpenTracing compatible implementation.
+
+## Configuration
+
+> **Note**: make sure that an `io.opentracing.Tracer` bean is available. It is not provided by this library.
+
+### Spring Boot
+Add the following starter dependency to your pom.xml:
+```xml
+<dependency>
+  <groupId>io.opentracing.contrib</groupId>
+  <artifactId>opentracing-spring-rabbitmq-starter</artifactId>
+</dependency>
+```
+
+### Spring
+Add the following dependency to your pom.xml:
```

Comment:
only this is needed? No other configuration is needed?

Reply:
Looks like nothing else is needed if single `RabbitTemplate` is used for all outbound AMQP messages (and `RabbitTemplate` bean exists), but when You'd like to use different RabbitTemplate for different outbound messages, then the AMQP integration turned out a hack, and i still have lots of pain in the @$$ (because of trying to re-use package-protected classes and methods from this library) ;)

Reply:
thanks @atsu85